### PR TITLE
Week 5 Objects and Arrays

### PR DESCRIPTION
**Pull request by yining1023**
Demo is here: [https://yining1023.github.io/ICM-Quizzes/week5-objects-arrays/](https://yining1023.github.io/ICM-Quizzes/week5-objects-arrays/).

The five questions are focus on:
1. Arrays
2. Arrays operations
3. Arrays of objects
4. The constructor function
5. Adding and removing objects
6. Clicking on objects

I didn't add the "checking objects intersection" or "images and objects" into the quiz. I think they might be little bit complicate and we cannot upload images to p5-widget. But I added them into the [editor examples](https://github.com/ITPNYU/ICM-2016/wiki/Future-Homework-Shiffman).

### CODE REVIEW

File: week5-objects-arrays/index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+  3. Write a program that implements the above sketch. Use an array called <code>bubbles</code> and a for loop to store and get access to each bubble object.</p>
+
+<script type="text/p5" data-preview-width="400">
+var bubbles = [];
+
+function setup() {
+  createCanvas(200, 200);
+  for (___________________) {
+    bubbles[____] = {
```
Comment (shiffman) [inline week5-objects-arrays/index.html]: we're not using this syntax anymore in ICM.  We are focusing on the constructor function only. So I would change this to maybe step 1 making a single object with a constructor function and then have the next one be the array of objects made with the constructor function.

**Review by shiffman — CHANGES_REQUESTED** [review]
Great, just the one change thank you!

**Comment by yining1023** [comment]
Got it! Just pushed the changes. :-)

**Comment by shiffman** [comment]
Awesome!

**Pull request merged**